dev/src/lobster/flat_gltf.h

```diff
@@ -0,0 +1,789 @@
+// This is free and unencumbered software released into the public domain.
+//
+// Anyone is free to copy, modify, publish, use, compile, sell, or
+// distribute this software, either in source code form or as a compiled
+// binary, for any purpose, commercial or non-commercial, and by any
+// means.
+//
+// In jurisdictions that recognize copyright laws, the author or authors
+// of this software dedicate any and all copyright interest in the
+// software to the public domain. We make this dedication for the benefit
+// of the public at large and to the detriment of our heirs and
+// successors. We intend this dedication to be an overt act of
+// relinquishment in perpetuity of all present and future rights to this
+// software under copyright law.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+// OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+// ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+// OTHER DEALINGS IN THE SOFTWARE.
+
+#pragma once
+#ifndef FLAT_GLTF_H
+#define FLAT_GLTF_H
+
+#ifndef FLATBUFFERS_IDL_H_
+#error Include flatbuffers/idl.h before flat_gltf.h so glTF models can be loaded.
+#endif
+
+#include <functional>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include "glincludes.h"
+
+#include "gltf_generated.h"
+
+// found in graphics.cpp
+extern Shader *currentshader;
+
+namespace fgl {
+
+bool verbose = true;
+
+#ifndef _VERBOUT_
+#define _VERBOUT_                                                              \
+  if (verbose)                                                                 \
+  std::cout
```

Comment:
We're not using `std::cout` anywhere in the lobster codebase. Please use LOG_ERROR, LOG_INFO and friends.